lib/init/settings.js

```diff
@@ -24,6 +24,20 @@ module.exports = (app) => {
   }).then(merged => {
     app.set('applications-merged', merged[0]);
 
+    return database('applications').select('*').orderBy('name', 'DESC').then(rows => {
```

Comment:
I fell in to this trap with APIs. We can't grab these only during initialization, we need to grab these when they're needed. Otherwise, if they change _after_ initialization, we won't have the updated list.